src/test/__tests__/ReactTestUtils-test.js

```diff
@@ -187,6 +187,17 @@ describe('ReactTestUtils', function() {
 
   });
 
+  it('Test scryRenderedDOMComponentsWithClass with className contains \\n', function() {
+    var renderedComponent = ReactTestUtils.renderIntoDocument(<div>Hello <span className={`x
+    y`}>Jim</span></div>);
+    var scryResults = ReactTestUtils.scryRenderedDOMComponentsWithClass(
+      renderedComponent,
+      'x'
+    );
+    expect(scryResults.length).toBe(1);
+
```

Comment:
done. I copied from "Test scryRenderedDOMComponentsWithClass with TextComponent", thought it's the appropriate  style ....